tests/application_tests/src/test_report.py

```diff
@@ -42,14 +42,19 @@ def __call__(self, driver):
 class OpenReportTest(unittest.TestCase):
     """Open a report."""
 
+    # Class names of MUI-components used in the tests
+    DASHBOARD_CARD_CLASS_NAME = "MuiCard-root"
+    DASHBOARD_CARD_HEADER_CONTENT_CLASS_NAME = "MuiCardHeader-content"
+
     def setUp(self):
         """Override to setup the driver."""
         firefox_options = webdriver.FirefoxOptions()
         firefox_options.add_argument("--headless")
         self.driver = webdriver.Remote("http://selenium:4444", options=firefox_options)
         self.driver.implicitly_wait(10)
         self.wait = WebDriverWait(self.driver, 10)
-        self.driver.get("http://www:8080?hide_toasts=true")  # Hide toasts so they don't block other visual elements
+        # Hide things so they don't block other visual elements
+        self.driver.get("http://www:8080?hide_toasts=true&hidden_cards=legend")
```

Comment:
Why `&hidden_cards=legend`? Was that a code change in this PR that I missed, or was it already something blocking other elements?

Reply:
I had some problems that I thought this would work around. I think this change can be reverted.